package.json

```diff
@@ -195,13 +198,15 @@
     "glob": "^7.1.6",
     "html-webpack-plugin": "^4.3.0",
     "http-server": "^0.9.0",
+    "husky": ">=6",
     "isomorphic-fetch": "^2.2.1",
     "jest": "^23.6.0",
     "jest-image-snapshot": "^4.0.2",
     "jsdoc": "^3.6.6",
     "jsdom": "^15.2.1",
     "json2csv": "^4.2.1",
     "jsonschema": "^1.1.1",
+    "lint-staged": ">=10",
```

Comment:
Do we need this if you're running it through `npx`?

Reply:
Probably not, but I just updated the pre-commit hook to match what they're using in the `components-library`, so now we do.

Reply:
perfect